src/pages/contact.tsx

```diff
@@ -13,8 +13,79 @@ import styles from "src/styles/contact.module.scss";
 
 // Data
 import { socials } from "src/data/socialData";
+import { Contacts, Severity } from "src/data/contactData";
 
 const Contact: React.FC = () => {
+  const emailRegex = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+  //Set alert display to none
+  const [alertInfo, setAlert] = useState(<Alert style={{ display: "none" }} />);
+  const [disableSubmit, setDisableSubmit] = useState(false);
+  const [contactDetails, setContactDetails] = useState<Contacts>({
+    firstname: "",
+    lastname: "",
+    email: "",
+    message: "",
+    subject: ""
+  });
+
+  const alertComponentValue = (text: string, type: Severity) => {
+    setAlert(
+      <Alert
+        severity={type}
+        variant="filled"
+        onClose={() => {
+          setAlert(<Alert style={{ display: "none" }} />);
+        }}
+      >
+        {text}
+      </Alert>
+    );
+  };
+
+  const handleChange = (event: any) => {
```

Comment:
Check out this [stack overflow post](https://stackoverflow.com/questions/40676343/typescript-input-onchange-event-target-value) to properly type event handlers.